sdk/python/feast/ui_server.py

```
@@ -7,10 +7,16 @@
 from fastapi import FastAPI, Response
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.staticfiles import StaticFiles
+from pydantic import BaseModel
 
 import feast
 
 
+class SaveDocumentRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)